components/anchor/index.tsx

```diff
@@ -85,26 +85,26 @@ export default class Anchor extends React.Component<AnchorProps, any> {
     }
   }
 
-  clickAnchorLink = (href, component) => {
+  clickAnchorLink = (href: string, component: HTMLElement) => {
     this._avoidInk = true;
     this.refs.ink.style.top = `${component.offsetTop + component.clientHeight / 2 - 4.5}px`;
     this.anchorHelper.scrollTo(href, this.props.offsetTop, getDefaultTarget, () => {
       this._avoidInk = false;
     });
   }
 
-  renderAnchorLink = (child) => {
-
-    const { href } = child.props;
-    if (child.type.__ANT_ANCHOR_LINK && href) {
-      this.anchorHelper.addLink(href);
-      return React.cloneElement(child, {
-        onClick: this.clickAnchorLink,
-        prefixCls: this.props.prefixCls,
-        bounds: this.props.bounds,
-        affix: this.props.affix || this.props.showInkInFixed,
-        offsetTop: this.props.offsetTop,
-      });
+  renderAnchorLink = (child: React.ReactElement<any>) => {
```

Comment:
I found a same name function in anchorlink.tsx, do not know what's the difference between them ?